airdrop/serverless.yml

```diff
@@ -678,3 +678,25 @@ functions:
               - cardanoEventConsumerQueue
               - Arn
           batchSize: 1
+
+  cardano-airdrop-transfer-claim-service:
+    warmup: true
+    handler: airdrop/application/handlers/airdrop_handlers.cardano_airdrop_transfer_claim_service
+    layers:
+      - ${file(./config.${self:provider.stage}.json):MplaceCommonPythonLib}
+      - ${file(./config.${self:provider.stage}.json):MplacePythonWeb3AndGrpc}
+    role: ${file(./config.${self:provider.stage}.json):ROLE}
+    vpc:
+      securityGroupIds:
+        - ${file(./config.${self:provider.stage}.json):SG1}
+        - ${file(./config.${self:provider.stage}.json):SG2}
+      subnetIds:
+        - ${file(./config.${self:provider.stage}.json):VPC1}
+        - ${file(./config.${self:provider.stage}.json):VPC2}
+    events:
+      - schedule:
+          rate: rate(1 hours)
+          name: ${file(./config.${self:provider.stage}.json):ENVIRONMENT}LoyaltyAirdropTransferClaimBatch
+          enabled: ${file(./config.${self:provider.stage}.json):loyalty_airdrop_cron_status}
+          input:
+            airdrop_id: ${file(./config.${self:provider.stage}.json):LoyaltyAirdropId}
```

Comment:
why do we need to provide airdrop_id here as a input?

Reply:
Because the service is generic.